src/backend/src/entities/player.entity.ts

```diff
@@ -2,26 +2,34 @@ import {
   Entity,
   Column,
   PrimaryGeneratedColumn,
-  OneToOne,
-  JoinColumn,
   ManyToOne,
   BaseEntity,
 } from 'typeorm';
 import { Game } from './game.entity';
-import { User } from './user.entity';
+
+// IDK, just placeholder values for now
+export enum EmotionIdEnum {
+  Calm,
+  Greedy,
+  NotGreedy,
+}
 
 @Entity()
 export class Player extends BaseEntity {
-  @PrimaryGeneratedColumn()
+  @PrimaryGeneratedColumn() // Do we even need this anymore?
```

Comment:
I think we could potentially use Id in the wait room logic, so yep we still need this.